lib/autobox/Transform.pm

```diff
@@ -74,6 +84,20 @@ particular when the values are hashrefs or objects.
 
     $books->uniq_by("id");
 
+    $books->order_by("name");
+    $books->order_by(name => "desc");
+    $books->order_by(price => "num");
+    $books->order_by(price => [ "num", "desc" ]);
+    $books->order_by(name => [ sub { uc($_) }, "desc" ]);
+    $books->order_by([ price_with_tax => $rate ] => "num");
```

Comment:
The first arg is the accessor syntax. It's an arrayref because it's a method with arguments.

That arrayref means map_by $_->price_with_tax($rate), and sort by that value, numerically.